playbook.md

```diff
@@ -971,189 +971,72 @@ to build an appropriate hosting platform for their service. This is fair
-example, technical clients may appreciate short, information-dense responses,
-while non-technical clients might perceive that style as rude or dismissive.
-
-In general:
-
-* We are personable, friendly and helpful
-* If things look like they're going to get difficult or the client seems unhappy, we are honest and assume good faith
-* If we screw something up, we take responsibility and apologise. If the client seems very upset, we let a delivery lead know.
-* If we do become annoyed or frustrated by a ticket, we respond later or speak to a delivery lead about reassigning it.
-
-##### Don't over-deliver
-
-Of course, every client would like us to go the extra mile to solve their
-problem. But they also understand that to do that for them would mean bad
-service for another client - or that we never get to their issue, because we're
-too busy gold-plating the solution to someone else's.
-
-While we do everything we can to make sure the client is happy with our
-solution, we are also mindful of what's practical. We don't do serious bits of
-development work on tickets, or trial new approaches. We don't play with new
-tools or sink hours into interesting bugs. We set those things aside, and do
-them later.
-
-The main purpose of a ticket is to take some action that solves the problem, as
-quickly as possible. Generally speaking, we do the most time-efficient thing
-that we can. Of several potential solutions that solve the problem, assuming
-none is a bad one, we do the one which can be implemented the most quickly.
+We work on tickets by having a developer assigned to our support team on a weekly rota. Developers work on tickets for the duration of their support week. If there is a ticket they need another developer to help with, this work is scheduled.
 
 #### Deciding what to work on
 
-You are free to work on whichever of your assigned tickets you think is most
-important. But there are some important things to bear in mind.
+Developers on support are free to work on whichever tickets they are assigned to and think is most important. But there are some important things to bear in mind.
 
 ##### Triage
 
 We all have limited time. We try to spend it wisely. All other things being
 equal, it is better to spend half an hour solving each of four tickets than to
 spend two hours on one issue.
 
-##### Procrastinating
-
-We don't procrastinate about dealing with difficult tickets. Most of our SLA
-breaches happen when a complex ticket arrives and is assigned to a busy person.
-In this situation, it's only natural that we prefer to do other work first. But
-it is vital that we don't let the hard tickets gather dust while we crack
-through the easy ones, so we try to be mindful of this bias.
-
 ##### Priority
 
 The priority of tickets is important, and we must be biased towards dealing with
 more urgent tickets before less urgent ones.
 
-Zendesk can help us to balance these factors. When you log in and land on the
```

Comment:
Just to check that this section's gone into the ops docs? It is useful info for people on support, and there's a section on workflow that this belongs to.

Reply:
Yes, the changes we're agreeing to remove from the playbook will be moved into the ops docs, once we know what we've decided to remove.